### PR TITLE
feat: Show notification after successfully updating a list

### LINKED ISSUE #1789: After editing a list a toast of success or error should appear to notify the user about the result

### PR DESCRIPTION
**Pull request by kevinszuchet**
Closes #1789

### CODE REVIEW

File: webapp/src/modules/toast/sagas.spec.ts [hunk 1]
```diff
@@ -46,13 +47,15 @@ import {
...
-  getDeleteListFailureToast
+  getDeleteListFailureToast,
+  getUpdateListSuccessToast
 } from '../toast/toasts'
 import { ItemBrowseOptions } from '../item/types'
 import { FetchNFTsFailureAction, fetchNFTsFailure } from '../nft/actions'
 import { List } from '../favorites/types'
 import { toastSaga } from './sagas'
 import { toastDispatchableActionsChannel } from './utils'
+import { UpdateOrCreateList } from '../vendor/decentraland/favorites/types'
```
Comment (flobarreto) [inline webapp/src/modules/toast/sagas.spec.ts:58]: can you move this import to line 55?

**Review by flobarreto — APPROVED** [review]

File: webapp/src/modules/toast/sagas.spec.ts (hunk 1 above)
Comment (LautaroPetaccio) [inline webapp/src/modules/toast/sagas.spec.ts:58]: Would you mind moving this import up?

File: webapp/src/modules/toast/sagas.spec.ts [hunk 2]
```diff
@@ -214,19 +217,19 @@ describe('when handling the failure of unpicking a favorite item', () => {
...
-    } as List
+    } as UpdateOrCreateList
   })
 
   it('should show a toast signaling the success of the list deletion', () => {
     return expectSaga(toastSaga)
       .provide([[select(getState), []]])
-      .put(showToast(getDeleteListSuccessToast(list), 'bottom center'))
-      .dispatch(deleteListSuccess(list))
+      .put(showToast(getUpdateListSuccessToast(list), 'bottom center'))
```
Comment (LautaroPetaccio) [inline webapp/src/modules/toast/sagas.spec.ts:231]: This test seems to be overwriting the delete one, can we create a separate one for the update?

**Review by LautaroPetaccio — COMMENTED** [review]
Looks great! 👍  I've left a comment about a test so we can review it.

**Review by LautaroPetaccio — APPROVED** [review]
🍟

**Pull request merged**